huaweicloud/services/cfw/resource_huaweicloud_cfw_protection_rule.go

```diff
@@ -646,6 +774,39 @@ func flattenGetProtectionRuleResponseBodyRuleIpRegionDto(resp interface{}) []int
 	return rst
 }
 
+func flattenGetProtectionRuleResponseBodyRuleTagsVO(resp interface{}) []map[string]interface{} {
+	curJson := utils.PathSearch("tag", resp, nil)
+
+	if curJson == nil {
+		return nil
+	}
+	tags := make([]map[string]interface{}, 0, 1)
```

Comment:
it is better to rename it to tag, because only one tag is suported

Reply:
fixed